conn_http.go

```diff
@@ -454,10 +467,48 @@ func (h *httpConnect) prepareRequest(ctx context.Context, reader io.Reader, opti
 		}
 		req.URL.RawQuery = query.Encode()
 	}
-
 	return req, nil
 }
 
+func (h *httpConnect) prepareRequest(ctx context.Context, query string, options *QueryOptions, headers map[string]string) (*http.Request, error) {
+	if options == nil || options.ignoreExternalTables || len(options.external) == 0 {
```

Comment:
```suggestion
	if options == nil || len(options.external) == 0 {
```

`ignoreExternalTables` looks redundant in this use case, if we properly pass the context here.

Reply:
just pushed an update to remove `ignoreExternalTables` in options